tools/flash_shelly.py

```diff
@@ -266,16 +266,16 @@ def write_flash(host, lfw, dlurl, cfw_type, mode):
     try:
       with urllib.request.urlopen(checkurl) as fp:
         info = json.load(fp)
+      if mode == 'homekit':
+        onlinecheck = info['version']
+      else:
+        onlinecheck = info['fw'].split('/v')[1].split('@')[0]
     except (urllib.error.HTTPError, urllib.error.URLError) as err:
       logger.debug(f"Error: {err}")
       n += 1
-    if info:
-      n=41
-    time.sleep(1)
-  if mode == 'homekit':
-    onlinecheck = info['version']
-  else:
-    onlinecheck = info['fw'].split('/v')[1].split('@')[0]
+    if onlinecheck == lfw:
+      n=16
```

Comment:
is this a weird way of saying `break` or am i missing something?

Reply:
ye, soz, was left over from debugging.